### PR TITLE
IstioOperator Helm chart for Kyma 2.0

### LINKED ISSUE #11636: Helm chart for istio istiooperator file
**Description**

Implement helm chart containing istiooperator CR file. This istio configuration should only contain changes kyma is introducing against default istio configuration. there should be a possibility to configure each value in istiooperator file. Create files for evaluation and producction profiles. 

**Reasons**

Simplify Istio installation / upgrade

**Attachments**

Part of https://github.com/kyma-project/kyma/issues/11635

### PR DESCRIPTION
**Pull request by cnvergence**
**Description**

Changes proposed in this pull request:

- Implement new istio chart containing minimal IstioOperator CR including only changes which kyma is introducing
- Support for profiles: production and evaluation

**Related issue(s)**

Fixes #11636

### CODE REVIEW

**Review by mjakobczyk — APPROVED** [review]

File: resources/istio-configuration/templates/tests/istio-kyma-validate.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{{- if .Capabilities.APIVersions.Has "testing.kyma-project.io/v1alpha1" }}
```
Comment (pbochynski) [inline resources/istio-configuration/templates/tests/istio-kyma-validate.yaml:1]: Do we need this? `helm template` will not render it.

File: resources/istio-configuration/templates/tests/istio-kyma-validate-rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (pbochynski) [inline resources/istio-configuration/templates/tests/istio-kyma-validate-rbac.yaml:1]: Do we need it?

**Review by pbochynski — CHANGES_REQUESTED** [review]

File: resources/istio-configuration/values.yaml [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+monitoring:
+  enabled: true
+  dashboards:
+    enabled: true
+  istioServiceMonitor:
+    enabled: true
+    scrapeInterval: ""
+
+test:
```
Comment (pbochynski) [inline resources/istio-configuration/values.yaml:116]: The test section can be removed.

**Review by pbochynski — APPROVED** [review]

**Pull request merged**